credentials/credentials.go

```diff
@@ -126,6 +126,10 @@ func (t TLSInfo) AuthType() string {
 	return "tls"
 }
 
+func (t TLSInfo) ConnState() tls.ConnectionState {
```

Comment:
The getter does not prohibit mutation. It is the same thing compared to making it exported.